apps/admin-x-framework/src/hooks/use-koenig-file-upload.ts

```
@@ -0,0 +1,236 @@
+
+    const fetchApi = useFetchApi();
+
+    function updateProgress() {
+        if (progressTracker.current.size === 0) {
+            setProgress(0);
+            return;
+        }
+
+        let totalProgress = 0;
+
+        progressTracker.current.forEach(value => totalProgress += value);
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)